Makefile

```diff
@@ -98,9 +98,7 @@ $(BUILD)/gfx/sgb/corrupted_b1e3.2bpp: tools/gfx += --trim-whitespace
 $(BUILD)/gfx/sgb/sgb_border_silver.2bpp: tools/gfx += --trim-whitespace
 
 $(BUILD)/gfx/trainer_card/leaders.2bpp: tools/gfx += --trim-whitespace
-
-$(BUILD)/gfx/pokegear/town_map.2bpp: tools/gfx += --trim-trailing
-
+$(BUILD)/gfx/trainer_gear/town_map.2bpp: tools/gfx += --trim-trailing
```

Comment:
It's nitpicking, but this could keep the blank lines separating it from unrelated gfx/\*/ subdirectory lines.

Reply:
Fixed.